frontend/public/locales/nl/translation.json

```diff
@@ -91,7 +91,9 @@
     "uploadError": "Project is niet goed geformatteerd",
     "noDeadlinesPlaceholder": "Nog geen opgegeven deadlines",
     "noFilesPlaceholder": "Nog geen opgave bestanden geupload",
-    "noRegexPlaceholder": "Nog geen regex toegevoegd"
+    "noRegexPlaceholder": "Nog geen regex toegevoegd",
+    "unauthorized": "U heeft niet de juiste rechten om een project aan te maken voor dit vak",
+    "submissionError": "Er is een fout opgetreden bij het indienen van uw project. Probeer het later opnieuw."
```

Comment:
You never use a . in other translations, in my opinion this would be better changed to " Er is een fout opgetreden bij het indienen van uw project, probeer het later opnieuw"

Reply:
Fixed in [044f50a](https://github.com/SELab-2/UGent-3/pull/313/commits/044f50aad59271d4c4305697af6ac45906c7d3ac)

👍 Valid remark 👍 💯 🔥